### ISSUE TITLE
Windows: unknown encoding: string-escape

### ISSUE THREAD

**Issue by NazBen** [wanglongqi/WriteTeX#10]
Hi,

First of all, thank you for the extension.

I followed the instructions and tested the extension with the simple `$\frac{1}{2}$` command but got this error:

```
Traceback (most recent call last):
  File "writetex.py", line 347, in <module>
    e.affect()
  File "C:\Program Files\Inkscape\share\extensions\inkex.py", line 283, in affect
    self.effect()
  File "writetex.py", line 164, in effect
    self.merge_pdf2svg_svg(svg_file)
  File "writetex.py", line 286, in merge_pdf2svg_svg
    WriteTexNS] = self.text.encode('string-escape')
LookupError: unknown encoding: string-escape

```

Any idea?

Thank you!

**Comment by wanglongqi** [comment]
What version of inkscape you are using? string-escape is an encoding in Python 2.

**Comment by NazBen** [comment]
Hi,

I'm using the "almost" last version: 0.92.1? I can upgrade to 0.92.2 to see how it goes.

**Comment by wanglongqi** [comment]
I never heard inkscape upgraded to python 3. You can try to change all "string-escape" in `writetex.py` into "unicode_escape".

**Comment by NazBen** [comment]
I did so and got this instead:
```
Traceback (most recent call last):
  File "writetex.py", line 347, in <module>
    e.affect()
  File "C:\Program Files\Inkscape\share\extensions\inkex.py", line 285, in affect
    self.output()
  File "C:\Program Files\Inkscape\share\extensions\inkex.py", line 272, in output
    self.document.write(sys.stdout)
  File "src\lxml\lxml.etree.pyx", line 2048, in lxml.etree._ElementTree.write (src\lxml\lxml.etree.c:65677)
  File "src\lxml\serializer.pxi", line 729, in lxml.etree._tofilelike (src\lxml\lxml.etree.c:139689)
  File "src\lxml\lxml.etree.pyx", line 324, in lxml.etree._ExceptionContext._raise_if_stored (src\lxml\lxml.etree.c:12150)
  File "src\lxml\serializer.pxi", line 646, in lxml.etree._FilelikeWriter.write (src\lxml\lxml.etree.c:138305)
TypeError: write() argument must be str, not bytes
```

**Comment by wanglongqi** [comment]
It seems the inkscape version you installed is using Python 3. I cannot reproduce this error on my machine. Maybe you can try to remove all `.decode('string-escape')`.

**Comment by NazBen** [comment]
Hi, 

Sorry for the late response. The problem was in the installation of Inkscape. Inkscape provides python 2.7 in the installation, but because I already had python I unchecked the installation of python with inkscape. Thus, when using WriteTex, it used my own python which was python 3...

I installed the latest version of Inkscape with its python interpretor and it works now.

Thanks for the help. Great extension!
